lib/constants.js

```diff
@@ -46,12 +46,30 @@ module.exports.ZOO_LOG_LEVEL_INFO = NativeZk.ZOO_LOG_LEVEL_INFO;
 /** @type {number} 4 */
 module.exports.ZOO_LOG_LEVEL_DEBUG = NativeZk.ZOO_LOG_LEVEL_DEBUG;
 
+/** @type {number} 0 */
+module.exports.ZOO_PERSISTENT = NativeZk.ZOO_PERSISTENT;
+
 /** @type {number} 1 */
 module.exports.ZOO_EPHEMERAL = NativeZk.ZOO_EPHEMERAL;
 
 /** @type {number} 2 */
 module.exports.ZOO_SEQUENCE = NativeZk.ZOO_SEQUENCE;
 
+/** @type {number} 2 */
+module.exports.ZOO_PERSISTENT_SEQUENTIAL = NativeZk.ZOO_PERSISTENT_SEQUENTIAL;
```

Comment:
Nice!

I was confused about the same number as ZOO_SEQUENCE, but[ found the origins](https://github.com/apache/zookeeper/blob/master/zookeeper-client/zookeeper-client-c/src/zookeeper.c#L126) of that.